frontend/src/citizen-frontend/App.tsx

```diff
@@ -58,18 +59,23 @@ export default function App() {
                       component={requireAuth(ApplicationReadView)}
                     />
                     {featureFlags.experimental?.incomeStatements && (
-                      <Route
-                        exact
-                        path="/income"
-                        component={requireAuth(IncomeStatements)}
-                      />
-                    )}
-                    {featureFlags.experimental?.incomeStatements && (
-                      <Route
-                        exact
-                        path="/income/:incomeStatementId"
-                        component={requireAuth(IncomeStatementEditor)}
-                      />
+                      <>
```

Comment:
I _think_ there have been some issues/bugs with `Route` under wrapping divs (i.e. not working). I'll see, if I can find an example.

Reply:
I used this pattern in messaging when it was enabled in staging only and do not recall having any issues. I can revert the change though.

Reply:
This is the issue I was talking about: https://github.com/remix-run/react-router/issues/5785
And it looks like it was never actually resolved, wonder how it worked with messaging...

Reply:
Weird. Well, I already reverted that change.